src/Query/Builder.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Linio\Component\Database\Query;
+
+class Builder
+{
+    public function flatParams(array $params): array
```

Comment:
Why need this method? 
Use currently only for `BuilderTest::testItDoesBuildFlatParamsWithNestedArrayValues`, not being used anywhere else.

Reply:
but, if is need, I think can be refactored to example like this:

    $flatParams = [];

    $iterator = new \RecursiveIteratorIterator(new \RecursiveArrayIterator($params));

    foreach($iterator as $value) {
         $flatParams[] = $value;
    }

    return $flatParams;

Reply:
good catch, that method was left behind after some refactoring. I just removed it. thanks.